recipes/liberrpa/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "LiberRPA" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/HUHARED/LiberRPA/archive/refs/tags/conda-forge-{{ version }}.tar.gz
+  sha256: 14a99881b545c2d14329ff43182e2983f819aec6f9fc8282e8b14ea88cc9b557
+
+  # Test locally: path: ../../../
+
+  # The setup.py is in "/condaLibrary", and setup.py will indicate the folder "/condaLibrary/liberrpa" as the source code folder. The folder argument (folder: condaLibrary/) does not work. use "cd condaLibrary" in "build-script"
+
+build:
+  # Skip (do not build) if not Windows OR Python < 3.12.
+  skip: True  # [not win or py<312]
+  script: cd condaLibrary && {{ PYTHON }} -m pip install . -vv
+  number: 2
+
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python =3.12
+    - ipykernel
+    - pywin32
+    - screeninfo
+    - pywinhook
+    - pyautogui
+    - pyqt
+    - pyqt-stubs
+    - pynput
+    - keyboard
+    - json5
+    - psutil
+    - pypdf
+    - pathvalidate
+    - pillow
+    - python-mss
+    - flask
+    - flask-socketio
+    - requests
+    - python-socketio
+    - websocket-client
+    - pyperclip
+    - sqlalchemy
+    - psycopg2
+    - pymysql
+    - pymssql
+    - oracledb
+    - pandas
+    - xlwings
+    - ftputil
+    - imapclient
+    - yagmail
+    - ffmpeg
+
+test:
+  imports:
+    - liberrpa
+
+about:
+  home: https://github.com/HUHARED/LiberRPA/tree/main/condaLibrary/liberrpa
+  summary: 'The main Python module of LiberRPA.'
+  description: |
+    The Python module is a main part of LiberRPA (a freeware that contains a series of RPA tools).
+    This package only works for Windows 10 or later, on x86-64 architecture.
+  license: AGPL-3.0-or-later
+  license_file: "LICENSE"
+  license_family: GPL3
```

Comment:
I am not sure if `AGPL-3.0-or-later` is actually `GPL3` license family. Can you explain this more, or just drop the `license_family` stuff?